### DISCUSSION TITLE
[Help]: Use embla without an bundler directly on a website

### DISCUSSION THREAD

**Discussion by janstieler** [davidjerleke/embla-carousel#697] [Help] [answered]
### Summary

Hi,
I have a project where I can't use a bundler. Is there a compiled version anywhere served?
I tried also the compiled CDN version. But here I got the Error `Cannot read properties of null (reading 'ownerDocument')`. I thought it should also run without CJS.
So have someone a solution?

Cheers

**Comment by davidjerleke** [comment, marked as answer]
@janstieler thanks for your question. This is the [get started guide](https://www.embla-carousel.com/get-started/cdn/) for usage with **CDN**. Reading the error message, my guess would be that you don't provide an element node to the `EmblaCarousel()` constructor. But it's impossible to tell without seeing your setup. 

- Here's a [working CodeSandbox](https://codesandbox.io/p/sandbox/embla-cdn-83xdwm?file=%2Findex.html) using CDN.

Best,
David

**Reply by davidjerleke** [reply]
@janstieler did you see [my response](https://github.com/davidjerleke/embla-carousel/discussions/697#discussioncomment-8143265)?

**Reply by janstieler** [reply]
Hi,
thanks for your help!
I get it to work.

Best
